muikku/pom.xml

```diff
@@ -253,6 +253,17 @@
       </dependencies>
     </profile>
 
+    <profile>
+      <id>matriculation-plugin</id>
+      <dependencies>
+        <dependency>
+          <groupId>fi.otavanopisto.muikku</groupId>
+          <artifactId>matriculation-plugin</artifactId>
+          <version>1.1.103-SNAPSHOT</version>
```

Comment:
Wrong version so it doesn't compile, please update to correct version which is 1.1.107-SNAPSHOT as set in correctly in matriculation-plugin/pom.xml